webui/react/src/pages/ExperimentDetails/CodeViewer/CodeViewer.tsx

```diff
@@ -418,6 +418,7 @@ const CodeViewer: React.FC<Props> = ({
     const hasActiveFile = activeFile?.text;
     const isSameFile = activeFile?.key === settings.filePath;
     const isIpybnFile = settings.filePath.includes('.ipynb');
+    // const isIpybnFile = /\.ipynb/i.test(settings.filePath);
 
     if (hasActiveFile && isSameFile && isIpybnFile) {
```

Comment:
I don't understand how this `if` condition works with one of the variables commented out

Reply:
oh I forgot to uncomment the new one during testing the rendering of the bad file. I'll fix this :)

Reply:
thanks good catch, it is good to go now